src/rdkafka_ssl.c

```diff
@@ -237,24 +237,21 @@ rd_kafka_transport_ssl_io_update(rd_kafka_transport_t *rktrans,
                         rd_kafka_ssl_error(NULL, rktrans->rktrans_rkb, errstr,
                                            errstr_size);
                 else if (!rd_socket_errno) {
-                        rd_rkb_dbg(rktrans->rktrans_rkb, BROKER, "SOCKET",
```

Comment:
I think we shouldn't remove these debug statements. I can see that we are logging these errstr in some case. Can we verify this? I would prefer "Disconnected" error change in a separate PR so that we can focus solely on those. I think we need to elaborate more on different kind of Disconnections which are very generic. We can brainstorm that in a separate PR.